### PR TITLE
Manage partials and images CEMS-2256

### PR DESCRIPTION
**Pull request by steve-giles**
This update provides the ability to manage partials and images.

images
- user is prevented from hot-rending the image and dragging and dropping data fields.
- can can select a local file which is rendered in the editor, base64 encoded

### CODE REVIEW

File: cypress/integration/Template_Details.feature [hunk 1]
```diff
@@ -10,7 +10,7 @@ Feature: View details of an existing template
     And Template details should contain a Body Uri "<body_uri>"
 
     Examples:
-    | row | template_id                            | body_uri                                       |
-    | 1   | 52602f9c-7547-11eb-b6f2-0242ac110003   | /template/52602f9c-7547-11eb-b6f2-0242ac110003 |
-    | 2   | bba87dc0-6583-11eb-a368-0242ac110003   | /template/bba87dc0-6583-11eb-a368-0242ac110003 |
+    | row | template_id                            | body_uri                                              |
+    | 1   | e854eedd-e0a3-4259-8fb2-4a126d52e715   | /api/v1/template/e854eedd-e0a3-4259-8fb2-4a126d52e715 |
+    | 2   | ea57e9dc-a203-11eb-871b-0242ac110003   | /api/v1/template/ea57e9dc-a203-11eb-871b-0242ac110003 |
```
Comment (danmcadams) [inline cypress/integration/Template_Details.feature:15]: These seem very specific to data that is already in the system. What happens if we change or delete some of these templates?

File: cypress/integration/Template_Grid_Search.feature [hunk 2]
```diff
@@ -23,7 +23,7 @@ Feature: Search list of existing templates
     Examples:
       | search_value    | num_of_templates |
       | badvalue        | 0                |
-      | Dan             | 8                |
+      | Dan             | 11               |
```
Comment (danmcadams) [inline cypress/integration/Template_Grid_Search.feature:26]: same here

**Review by danmcadams — APPROVED** [review]
The only issues I see are the very specific template IDs and names. Since we're using real data on a test server, this could cause issues for your tests in the future. I setup a mock server at one point with postman. We can probably update that and you can use that data instead. Or, if you know this will work for you, keep this.

File: cypress/integration/Template_Grid_Search.feature (hunk 2 above)
Comment (steve-giles) [reply cypress/integration/Template_Grid_Search.feature:26]: same above

File: cypress/integration/Template_Details.feature (hunk 1 above)
Comment (steve-giles) [reply cypress/integration/Template_Details.feature:15]: Yes, that's a good point, and if data changes the test will fail.  I hate to mock the data.  Would be nice to have a server dedicated just to regression testing, but that may not be feasible.  I may have to revert to mocking the data...

**Pull request merged**